src/main/resources/templates/repo/elasticbeanstalk-template.json.vpt

```diff
@@ -362,7 +362,17 @@
 						"Namespace": "aws:elasticbeanstalk:application:environment",
 						"OptionName": "org.sagebionetworks.stack.repo.beanstalk.number",
 						"Value": "${repoBeanstalkNumber}"
-					}
+					},
+                    {
+                        "Namespace": "aws:elasticbeanstalk:cloudwatch:logs",
+                        "OptionName": "StreamLogs",
+                        "Value": "true"
+                    },
+                    {
+                        "Namespace": "aws:elasticbeanstalk:cloudwatch:logs",
+                        "OptionName": "DeleteOnTerminate",
+                        "Value": "true"
+                    }
```

Comment:
Make it false and another 2 weeks (so retention 3 weeks) in case we need to look at the logs of a deleted stack.